examples/oed_utils.jl

```diff
@@ -237,6 +249,50 @@ function build_start_point(A, N, ub)
     return x, active_set, S
 end
 
+"""
+Build domain feasible for any node.
```

Comment:
improve the docstring sentence maybe to make it more complete

Reply:
this is especially important since this is the main example presenting this feature